mrpt_graphslam_2d/CMakeLists.txt

```diff
@@ -18,3 +18,47 @@ if (BUILD_TESTING)
     ${test_output_directory}/${PROJECT_TEST_NAME})
 endif()
 
+set(PROJECT_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../)
+
+#V-REP Config
+################################################################################
+
+if (EXISTS "$ENV{VREP}")
+  message(STATUS "Found V-REP")
+  set (V-REP_ROOT "${PROJECT_ROOT_DIR}/../" CACHE "V-REP root dir" STRING)
```

Comment:
- No dashed variables.
- I don't understand what you are doing here. You don't use $ENV{VREP} but instead set a V-REP_ROOT variable afterwards